evalai/utils/auth.py

```diff
@@ -76,6 +76,37 @@ def get_user_auth_token():
         sys.exit(1)
 
 
+def validate_user_auth_token_by_profile(auth_token):
```

Comment:
Sorry for the delayed review. @takitsuse Can you please change the name here to -
`validate_user_auth_token_by_profile` -> 'is_auth_token_valid`

Reply:
This function should return a boolean value if the request passes.